frontend/yarn.lock

```diff
@@ -0,0 +1,6518 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@jerboaa Could this file be removed? It seems to be a build time thing...

Reply:
@Karm While this is auto-generated, I still think we should keep it in order to maintain deterministic builds. See https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

Basically, an update to the file means an update to some dependency chain. But it would get updated by `yarn` so no trouble to check it in, IMO.